generated/nidaqmx/nidaqmx.proto

```diff
@@ -17,13 +17,40 @@ import "session.proto";
 
 service NiDAQmx {
   rpc ClearTask(ClearTaskRequest) returns (ClearTaskResponse);
+  rpc CreateAIVoltageChan(CreateAIVoltageChanRequest) returns (CreateAIVoltageChanResponse);
+  rpc CreateDIChan(CreateDIChanRequest) returns (CreateDIChanResponse);
+  rpc CreateDOChan(CreateDOChanRequest) returns (CreateDOChanResponse);
   rpc CreateTask(CreateTaskRequest) returns (CreateTaskResponse);
+  rpc StartTask(StartTaskRequest) returns (StartTaskResponse);
+  rpc StopTask(StopTaskRequest) returns (StopTaskResponse);
 }
 
 enum NiDAQmxAttributes {
   NIDAQMX_ATTRIBUTE_UNSPECIFIED = 0;
 }
 
+enum InputTermCfgWithDefault {
+  INPUT_TERM_CFG_WITH_DEFAULT_UNSPECIFIED = 0;
+  INPUT_TERM_CFG_WITH_DEFAULT_Cfg_Default = -1;
+  INPUT_TERM_CFG_WITH_DEFAULT_RSE = 10083;
+  INPUT_TERM_CFG_WITH_DEFAULT_NRSE = 10078;
+  INPUT_TERM_CFG_WITH_DEFAULT_Diff = 10106;
+  INPUT_TERM_CFG_WITH_DEFAULT_PseudoDiff = 12529;
+}
+
+enum LineGrouping {
+  option allow_alias = true;
+  LINE_GROUPING_UNSPECIFIED = 0;
+  LINE_GROUPING_ChanForAllLines = 1;
+  LINE_GROUPING_ChanPerLine = 0;
+}
+
+enum VoltageUnits2 {
```

Comment:
Note: this name is used in both the DAQ header and the CVI metadata files. There is a VoltageUnits1 but we're not using it yet.